### PR TITLE
Add installation instruction for Ubuntu

### PR DESCRIPTION
**Pull request by JonMagon**
https://github.com/crow-translate/crow-translate.github.io/pull/1

### CODE REVIEW

File: content/en/home/installation.md [hunk 1]
```diff
@@ -101,3 +101,13 @@ sudo zypper install crow-translate
 ```bash
 sudo eopkg it crow-translate
 ```
+
+#### Ubuntu, Linux Mint, KDE Neon, etc
+
+:package: [Personal Package Archive Ubuntu](https://launchpad.net/~jonmagon/+archive/ubuntu/crow-translate)
```
Comment (Shatur) [inline content/en/home/installation.md]: Maybe just use `PPA` in all languages? Sounds more familiar to me. What do you think?
And maybe put this right after Arch and Manjaro? These are very popular distributions.
```suggestion
:package: [PPA](https://launchpad.net/~jonmagon/+archive/ubuntu/crow-translate)
```

Comment (JonMagon) [reply content/en/home/installation.md]: Yeah, good idea. I want to mention PPAs are hosted in launchpad, doesn't `Launchpad PPA` sound better?

Comment (Shatur) [reply content/en/home/installation.md]: Yes, I agree :)

**Comment by Shatur** [comment]
Thank you a lot!

**Pull request merged**